src/activeobjectmgr.h

```diff
@@ -32,14 +35,30 @@ class ActiveObjectMgr
 	friend class ::TestServerActiveObjectMgr;
 
 public:
+	ActiveObjectMgr() = default;
+	DISABLE_CLASS_COPY(ActiveObjectMgr);
```

Comment:
What about move constructors?

Reply:
There is no implicit default move constructor, or move assignment operator if the copy constructor is deleted.